tv_apps/fling-player/js/video_player.js

```diff
@@ -1,29 +1,30 @@
-/* global evt */
-
 (function(exports) {
   'use strict';
 
-  function $(id) {
-    return document.getElementById(id);
-  }
-
-  function VideoPlayer() {
+  /**
+   * This class is a thin wrapper of HTMLVideoElement
+   * @param {HTMLVideoElement}
+   */
+  function VideoPlayer(player) {
+    this._player = player;
   }
 
-  var proto = evt(VideoPlayer.prototype);
+  var proto = VideoPlayer.prototype;
 
   proto.init = function vp_init() {
-    this._player = $('player');
     this._player.mozAudioChannelType = 'content';
 
+    this._isLoaded = false;
+    this._isPlaying = false;
+
     this._player.addEventListener('loadedmetadata', this);
-    this._player.addEventListener('seeked', this);
-    this._player.addEventListener('waiting', this);
     this._player.addEventListener('playing', this);
-    this._player.addEventListener('timeupdate', this);
     this._player.addEventListener('pause', this);
     this._player.addEventListener('ended', this);
```

Comment:
Can these events be heard by `FlingPlayer`? I saw you deleted all the event firing.